lib/rbvmomi/connection.rb

```diff
@@ -195,8 +195,7 @@ def obj2xml xml, name, type, is_array, o, attrs={}
       end
       xml
     rescue
-      $stderr.puts "#{$!.class} while serializing #{name} (#{type}):"
-      PP.pp o, $stderr
+      RbVmomi.logger.error("#{$!.class} while serializing #{name} (#{type}): #{o.pretty_inspect}")
```

Comment:
Technically, the original had a newline here...it's up to you.
```suggestion
      RbVmomi.logger.error("#{$!.class} while serializing #{name} (#{type}):\n#{o.pretty_inspect}")
```

Reply:
Or maybe 2 separate logger.error calls?

Reply:
Yeah we can include a newline, prefer to not do 2 separate calls as that makes it harder to find in a log tool like kibana